website/content/docs/agent/telemetry.mdx

```diff
@@ -684,3 +684,22 @@ agent. The table below describes the additional metrics exported by the proxy.
 | `consul.proxy.web.upstream.conns`   | Shows the current number of connections open from a proxy instance to an upstream. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to. | connections | gauge   |
 | `consul.proxy.web.inbound.rx_bytes` | Increments by the number of bytes received from an upstream connection. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to.       | bytes       | counter |
 | `consul.proxy.web.inbound.tx_bytes` | Increments by the number of bytes transferred to an upstream connection. Where supported a `src` label is added indicating the service name the proxy represents, and a `dst` label is added indicating the service name the upstream is connecting to.      | bytes       | counter |
+
+## Peering metrics
+
+** Requirements: **
+* Consul 1.13.0+
+
+Consul 1.13 introduced [cluster peering](/docs/connect/cluster-peering). These metrics are only emitted by the leader server.
+
+| Metric                                | Description                                                           | Unit   | Type    |
+| ------------------------------------- | ----------------------------------------------------------------------| ------ | ------- |
+| `consul.peering.exported_services`    | Counts the number of services exported to a peer cluster.  | count  | gauge   |
+
+### Labels
+
```

Comment:
```suggestion
Consul attaches the following labels to metric values. 
```
I thought an introduction might be helpful. Is that how labels for metrics work? Or are labels attributes that the user can assign? Is the concept of labels for metrics described anywhere?

Reply:
no labels are not user controlled

Reply:
and I am not entirely sure that our docs explain label based metrics anywhere 🤔  (at least I couldn't find a link right now)